members/models/activity.py

```diff
@@ -93,3 +94,23 @@ def participants(self):
         return self.activityparticipant_set.count()
 
     participants.short_description = "Deltagere"
+
+    def clean(self):
+        errors = {}
+        min_amount = 0
+
+        if self.activitytype.id == "FORENINGSMEDLEMSKAB":
+            min_amount = 75
+
+        if self.activitytype.id == "FORLØB":
+            min_amount = 100
+
+        if self.price_in_dkk < min_amount:
+            errors["price_in_dkk"] = (
+                "Prisen er for lav. Denne type aktivitet skal mindst koste "
+                + str(min_amount)
+                + " kr."
```

Comment:
Minor: prøvede med f-string lokalt, og det virker. Giver lidt enklere kode, og slipper for `str()` konvertering. Du vælger om du vil ændre, nuværende er også fint nok
```suggestion
                f"Prisen er for lav. Denne type aktivitet skal mindst koste {min_amount} kr."
```